apps/Makefile.in

```diff
@@ -92,10 +93,14 @@ install:
 	 cp $$i $(DESTDIR)$(OPENSSLDIR)/misc/$$i.new; \
 	 chmod 755 $(DESTDIR)$(OPENSSLDIR)/misc/$$i.new; \
 	 mv -f $(DESTDIR)$(OPENSSLDIR)/misc/$$i.new $(DESTDIR)$(OPENSSLDIR)/misc/$$i ); \
-	 done
-	@cp openssl.cnf $(DESTDIR)$(OPENSSLDIR)/openssl.cnf.new; \
-	chmod 644 $(DESTDIR)$(OPENSSLDIR)/openssl.cnf.new; \
-	mv -f  $(DESTDIR)$(OPENSSLDIR)/openssl.cnf.new $(DESTDIR)$(OPENSSLDIR)/openssl.cnf
+	 done;
+	@set -e; for i in $(CONFS); \
+	do \
+	(echo installing $$i; \
+	 cp $$i $(DESTDIR)$(OPENSSLDIR)/$$i.new; \
+	 chmod 644 $(DESTDIR)$(OPENSSLDIR)/$$i.new; \
+	 mv -f $(DESTDIR)$(OPENSSLDIR)/$$i.new $(DESTDIR)$(OPENSSLDIR)/$$i ); \
+	 done;
```

Comment:
Ah they're all redundant? In that case, I'll remove them all (that is, the ones following "done").